### PR TITLE
El 3668 select custom icon

### PR DESCRIPTION
**Pull request by dermot-hughes-mf**
#### Checklist

* [x] The contents of this PR are mine to submit. No third party code or other material is being committed.
* [x] New third party dependencies (if any) are linked via `package.json`. Third party dependencies are licenced under one of the following: Apache, MIT, BSD, Mozilla Public License, Eclipse Public License, or Oracle Binary Code License.
* [x] The code in this PR does not contain export-restricted encryption algorithms.

#### Ticket / Issue

https://portal.digitalsafe.net/browse/EL-3668

#### Description of Proposed Changes

Updated test and screenshot to move back to top of screen.

#### Documentation CI URL

https://pages.github.houston.softwaregrp.net/sepg-docs-qa/UXAspects_CI_UXAspects_El-3668-select-custom-icon

### CODE REVIEW

File: e2e/tests/components/select/standard/select.e2e-spec.ts [hunk 1]
```diff
@@ -538,7 +538,7 @@ describe('Select Tests', () => {
         expect(await imageCompare('select-tag-overflow')).toEqual(0);
     });
 
-    it('should allow a custom icon', async () => {
+    fit('should allow a custom icon', async () => {
```
Comment (donna-marijne) [inline e2e/tests/components/select/standard/select.e2e-spec.ts]: Can you just revert this change?

Comment (dermot-hughes-mf) [reply e2e/tests/components/select/standard/select.e2e-spec.ts]: Whoops! Reverted.

**Comment by donna-marijne** [comment]
Also just a suggestion, try to make sure to make a new branch in git for each PR.
